pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2642,6 +2649,27 @@ func maybeResizeDisk(diskfile string, maxsizebytes uint64) error {
 	return err
 }
 
+// Fetch the list of environment variables from the cloud init
+// We are expecting the environment variables to be pass in particular format in cloud-int
```

Comment:
Can you also include the fact that the entire cloud-init data string is treated as a set of environment variables - no keywords to parse a separate section from it?